internal/bminventory/inventory_test.go

```diff
@@ -5707,6 +5715,21 @@ var _ = Describe("TestRegisterCluster", func() {
 		})
 		Expect(reply).Should(BeAssignableToTypeOf(common.NewApiError(http.StatusBadRequest, errors.Errorf("error"))))
 	})
+
+	It("openshift release image and version successfully defined", func() {
+		mockClusterRegisterSuccess(bm, true)
+		reply := bm.RegisterCluster(ctx, installer.RegisterClusterParams{
+			NewClusterParams: &models.ClusterCreateParams{
+				Name:             swag.String("some-cluster-name"),
+				PullSecret:       swag.String(`{\"auths\":{\"cloud.openshift.com\":{\"auth\":\"dG9rZW46dGVzdAo=\",\"email\":\"coyote@acme.com\"}}}"`),
+				OpenshiftVersion: swag.String(common.TestDefaultConfig.OpenShiftVersion),
```

Comment:
You didn't test providing ReleaseImage

Reply:
the ReleaseImage should be already in the version (i.e. we add it as part of AddOpenshiftVersion)